### ISSUE TITLE
server: bind failed with error: 10048 

### ISSUE THREAD

**Issue by johnkubik** [JuicyPasta/Claymore-No-Fee-Proxy#2]
I get the error server: bind failed with error: 10048 (port for remote management is busy, use different -mport value), next attempt in 20sec...

I am using windows and configured it as you said.

Any ideas?

**Comment by drdada** [comment]
Did you get the error on the no-fee-proxy window ?
It seems the port is already taken by the remote management engine.
Can you change the proxy port ? or change the claymore's remote port (with -mport argument)

**Comment by johnkubik** [comment]
Yes, the error was in the proxy window. I can try and change one, and then the other and let you know.

This is just an issue on one of my rigs. The other rigs get different issues.

There are some weird connection issues with my other boxes as well. Most of the time they revert to the epools.txt. I can tell because I set a different name in the epools.txt so I can see in the pool display if they failed over.

They everyone one of my boxes fails over at least once an hour.

Any idea?

**Comment by JuicyPasta** [comment]
Are you using ethos by chance?

**Comment by johnkubik** [comment]
No, this is all on Windows.

I would be glad to help trouble shoot all of the issues I am experiencing. This is such a great project and I would like to help work the kinks out.

What can I do to assist?

**Comment by JuicyPasta** [comment]
I added you as a contributor, feel free to make changes as you see fit. I have some time to debug this tomorrow, also feel free to dm me in a gitter or something.

**Comment by drdada** [comment]
@johnkubik Do you still have the issue with the port ? Try to change and disable the remote management engine with -mport 0
For the epool.txt issue, i do no use this file, but the pool failover have to be disabled.

**Comment by Ravaelles** [comment]
I've had this issue and good ol' restart helped.

**Comment by seagemgames** [comment]
adding -mport 0 fixed it for me :D

**Comment by dporter536** [comment]
restart sorted me aswell
